### ISSUE TITLE
--shodan_query flag use cases

### ISSUE THREAD

**Issue by Celestial-intelligence** [ztgrace/changeme#83]
how to make more complicated --shodan_query (for example http.title:"VIDEO WEB SERVER" port:80 country:US) while it takes only one argument? 
or how to get more than 100 results when using shodan_query?

**Comment by ztgrace** [comment]
Yes, the shodan `search` function takes a `page=X` kwarg. I feel the most reasonable way to implement this would be to add an option to specify the number of pages with an option for `all`. Something like `--shodan_pages=10` or `--shodan_pages=all`.
